google/generativeai/protos.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module publishes the ProtoBuffer "Message" classes used by the API.
+
+ProtoBufers are Google API's serilization format. They are strongly typed and efficient.
+
+The `genai` SDK tries to be permissive about what objects it will accept from a user, but in the end
+the SDK always converts input to an appropriate Proto Message object to send as the request.
+
+If you have any uncertainty about what the API may accept or return, these classes provide the
+complete/unambiguous answer. They come from the `google-ai-generativelanguage` package which is
+generated from a snapshot of the API definition.
+
+>>> from google.generativeai import protos
+>>> import inspect
+>>> print(inspect.getsource(protos.GenerateContentRequest))
```

Comment:
Any reason for choosing `GenerateContentRequest`? It's not exposed to the user in any way so might be confusing.

Reply:
Replaced with `Part`.